sources/packages/api/src/services/application/application.service.ts

```diff
@@ -317,8 +317,34 @@ export class ApplicationService extends RecordDataModelService<Application> {
   ): Promise<Application> {
     const application = await this.repo
       .createQueryBuilder("application")
+      .select([
+        "application.data",
+        "application.id",
+        "application.applicationStatus",
+        "application.pirStatus",
+        "application.assessmentStatus",
+        "application.coeStatus",
+        "application.applicationStatusUpdatedOn",
+        "application.applicationNumber",
+        "offering",
+        "location",
+        "programYear",
+        "coeDeniedReason.id",
+        "coeDeniedReason.reason",
+        "pirDeniedReasonId.id",
+        "pirDeniedReasonId.reason",
+        "application.coeDeniedOtherDesc",
+        "application.pirDeniedOtherDesc",
+      ])
+      .leftJoin("application.offering", "offering")
+      .leftJoin("application.location", "location")
+      .leftJoin("location.institution", "institution")
+      .leftJoin("institution.institutionType", "institutionType")
       .leftJoin("application.student", "student")
       .leftJoin("student.user", "user")
```

Comment:
Hi @ann-aot , I know this is not part of your change, but @andrewsignori-aot @abschwenker @dheepak-aot @ann-aot what you think of having a innerjoin for the user and student joins. As we are fetching with respect to userId and the user should be a student. Please share your thoughts